projects/rpx-xui-translation/src/lib/rpx-translation.service.ts

```diff
@@ -109,8 +108,9 @@ export class RpxTranslationService {
             this.phrases[phrase].next(`${phrase} [Translation in progress]`);
             this.load(phrase, lang, yesOrNo);
           }
-        })
-      ).subscribe(() => { });
+        }),
+        pipe(take(1))
```

Comment:
@MateiHMCTS - any reason for the double `pipe`? Isn't there already one from Line 99?

Reply:
Forgot to remove it